HISTORY.rst

```diff
@@ -25,6 +25,60 @@ unreleased
   are supported. Once registered, the device will show up in the linked IoT Hub and can be interacted with or
   simulated using other `az iot device` commands.
 
+**IoT Central updates**
+
+* Add support for device groups CRUD.
+  - az iot central device-group
+    - az iot central device-group list
+    - az iot central device-group show
+    - az iot central device-group create
+    - az iot central device-group delete
+    - az iot central device-group update
+
+* Add support for device attestation CRUD.
+  - az iot central device attestation
+    - az iot central device attestation show
+    - az iot central device attestation create
+    - az iot central device attestation delete
+    - az iot central device attestation update
+
+* Add support for device/module properties/telemetry/command.
+  - az iot central device list-components
+  - az iot central device telemetry
+    - az iot central device telemetry show
+  - az iot central device component-telemetry
+    - az iot central device component-telemetry show
+  - az iot central device properties
+    - az iot central device properties show
+    - az iot central device properties replace
+    - az iot central device properties update
+    - az iot central device component-properties show
+    - az iot central device component-properties replace
+    - az iot central device component-properties update
+  - az iot central device edge module list-components
```

Comment:
Shouldn't change any thing from ```az iot central device edge```, the concept `module` here is more on the agent definition within deployment manifest, we will have manifest associated, update operation for this command group soon.
all twin, command, telemetry from edge device should follow the same group as ``` az iot central device command|twin|telemetry```

Reply:
Yes. I'm updating these commands. Thanks for explanations.